src/common/variables.scss

```diff
@@ -2,6 +2,8 @@
 $primary: rgba(173, 216, 230, 0.5);
 $secondary: rgba(128, 128, 128, 1);
 $tertiary: rgba(255, 165, 0, 0.5);
+$temperature: red;
+$wind: blue;
```

Comment:
Personally I'd go with $primary-color, $temperature-color,  etc. so that I have more context when using it

Reply:
Agree. Try to see if this approach suits you:
1) create base colors, say, 
```
$green_light: #xxx; 
$green_regular: #yyy;
$green_dark: #zzz;

$blue_light: #vvv;
...
```

2) create (if you need) abstract colors reflecting your current brand palette:
```
$primary-color: $green_light;
$secondary-color: $blue_dark;

$font-color: $primary-color (ideally) || (may optionally be) $green_color;
...
```

3) use your colors in places. Come up with meaningfull names:
```
$wind-icon-color: $primary-color || $font-color.
$temperature-icon-color: $secondary-color
```

note how '_' are used to define low level variables. and '-' is used to defien highly abstracted varables. That will also help later to distinguish between those.

Optionally you may omit using primary, secondary... that depends on consistency of your designs. That would be your desision. But the point is whenever you want to change everything from being green to blue - you just change primary color value. You will not have to go all ober the app and change all specific colors manually.